iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/unix_domain_socket.hpp

```diff
@@ -140,9 +140,11 @@ class UnixDomainSocket : public DesignPattern::Creation<UnixDomainSocket, IpcCha
     static constexpr int32_t INVALID_FD = -1;
 
     UdsName_t m_name;
-    IpcChannelSide m_channelSide;
+    IpcChannelSide m_channelSide{};
```

Comment:
Please initialize it with an explicit enum class value like `IpcChannelSide m_channelSide = IpcChannelSide::CLIENT;`.